plugins/scenario/World.ts

```diff
@@ -22,13 +22,14 @@ export class World {
     // Q: should we really need to fork/snapshot the deployment manager?
     this.hre = hre;
     this.base = base;
-    this.deploymentManager = new DeploymentManager(base.name, hre, { debug: true });
+    const networkName = base.name === 'development' ? 'development' : base.network;
```

Comment:
`this.isRemoteFork() ? base.network : 'development';`?

Reply:
that works. updated

Reply:
deleted; we settled on using `deployments/hardhat`